changelog/4108.bugfix.rst

```diff
@@ -0,0 +1 @@
+Resolve symbolic links for args.
```

Comment:
I think it is better to explain better what this is solving and what are the consequences of this.

Reply:
Ok, will add a paragraph below it, once this turns out to be green finally.

Reply:
Is this fine?  (from the commit message)

> This fixes running `pytest tests/test_foo.py::test_bar`, where `tests`
is a symlink to `project/app/tests`: previously
`project/app/conftest.py` would be ignored for fixtures then.

Reply:
Sure, that will do. 👍 Thanks!

Reply:
Cool, amended.